package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "testcafe-browser-natives",
-  "version": "0.11.0",
+  "version": "0.12.0",
```

Comment:
This is breaking change, so we should update the `major` number. The removed `resize` function is used in [TestCafe](https://github.com/DevExpress/testcafe/blob/master/src/test-run/index.js#L326)